releasenotes/notes/lattice-from-networkx-20c7c8119af77f36.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
```

Comment:
I'm wondering whether we should split out the edge type validation fix into another PR which could become part of a `0.3.1` bugfix release. If it get's included in here we will need to wait for the `0.4.0` feature release.

Reply:
It's probably not very urgent since it might be expected that putting some random values on the edges breaks somewhere down the stack. But of course it's nice if it breaks early and with a clear warning, I can separate it out if you like!

Reply:
As you see fit. I'm just seeing now that the actual comment I wanted to write got somehow lost :thinking: 
Cause I wanted to say that this fix should be listed separately in the reno :sweat_smile: